syncd/Syncd.cpp

```diff
@@ -4023,6 +4027,23 @@ void Syncd::snoopGetOid(
         return;
     }
 
+    /*
+    * Check if object was previously discovered on this switch, then no need to update ASIC_STATE.
+    */
+    if (!isInitViewMode())
+    {
+        sai_object_id_t rid = SAI_NULL_OBJECT_ID;
+        if (m_translator->tryTranslateVidToRid(vid, rid))
+        {
+            const auto switchVid = VidManager::switchIdQuery(vid);
+            if (m_switches[switchVid]->isDiscoveredRid(rid))
+            {
+                // Already discovered object.
+                return;
+            }
+        }
```

Comment:
add empty line before each if, also since rid is out paremeter, you dont need to initialize it

Reply:
done